APIhomework/src/index.js

```diff
@@ -29,16 +33,9 @@ function beginSearch(event) {
   city.innerHTML = `${searchInput.value.trim().toUpperCase()}`;
   let apiK = "fb8b95424c106907f53c4fc0092c4971";
   let apiUrl = `https://api.openweathermap.org/data/2.5/weather?q=${searchInput.value}&units=imperial&appid=${apiK}`;
-  axios.get(apiUrl).then(showTemperatureByCity);
-}
-
-function showTemperatureByCity(response) {
-  let temperature = Math.round(response.data.main.temp);
-  let currentTemp = document.querySelector(".currently-temp");
-  currentTemp.innerHTML = `${temperature}°F`;
-  let country = document.querySelector(".currently-country");
-  let countryName = response.data.sys.country;
-  country.innerHTML = `${countryName}`;
+  axios.get(apiUrl).then(function(response) {
+    showTemperature(response.data.main.temp, response.data.name, response.data.sys.country);
```

Comment:
Again we just need to call the `showTemperature` function here with the correct response parameters.
Technically you've already set the city text above, but it doesn't hurt to replace it with the response back from the API, since that will technically be more accurate that the user input anyway.